TypeCobol/Compiler/CompilationDocument.cs

```diff
@@ -149,15 +151,14 @@ public void OnStopRefreshParsingStep()
         /// This method does not scan the inserted text lines to produce tokens.
         /// You must explicitely call UpdateTokensLines() to start an initial scan of the document.
         /// </summary>
-        public CompilationDocument(TextSourceInfo textSourceInfo, IEnumerable<ITextLine> initialTextLines,
-            TypeCobolOptions compilerOptions, IProcessedTokensDocumentProvider processedTokensDocumentProvider, List<RemarksDirective.TextNameVariation> copyTextNameVariations) :
-            this(textSourceInfo, initialTextLines, compilerOptions, processedTokensDocumentProvider, null, copyTextNameVariations)
+        public CompilationDocument(bool isCopy, bool isImported, TextSourceInfo textSourceInfo, IEnumerable<ITextLine> initialTextLines, TypeCobolOptions compilerOptions, IProcessedTokensDocumentProvider processedTokensDocumentProvider,
```

Comment:
Not exactly, `isImported` means that the source is included in another document through a `COPY` or `EXEC SQL INCLUDE` instruction. It cannot be used with `isCopy` set to `false` so in a sense it also means "use direct parsing" but from the caller point of view, I thought it would be clearer to have the two parameters.

Reply:
Done (968a5e72f3f656d2fa21a7b4c388457143350a5a)